services/catalog/src/simcore_service_catalog/exceptions/handlers/__init__.py

```diff
@@ -6,8 +6,8 @@
 
 
 def setup_exception_handlers(app: FastAPI) -> None:
-    app.add_exception_handler(HTTPException, http_error_handler)
```

Comment:
Not necessary to ignore, you can change the function signature as shown here to shut-up mypy: https://github.com/ITISFoundation/osparc-simcore/blob/05cff9e1dd039bb28c7cb37c1b98210e9d409715/services/api-server/src/simcore_service_api_server/exceptions/handlers/_http_exceptions.py#L8